### ISSUE TITLE
[QUESTION] Debugging: Cannot find module 'typeorm'

### ISSUE THREAD

**Issue by deaktomi** [TypedProject/tsed#799]
- I'm following your tutorials.
- I created a project and works like a charm. I can "serve" it by `yarn start`. Or I can debug it in Visual Studio Code by pressing F5.
- Now I added TypeOrm to my project. By following [this tutorial](https://tsed.io/tutorials/typeorm.html#use-entity-typeorm-with-controller)
- It works fine with `yarn start` but it is **not working with VSCode debug**. I get the following errors:
```
Cannot find module 'typeorm'
Cannot find module '@tsed/typeorm'
```
In some other cases, I had to add '@types/<package_name>' to my devDependencies. For example, I have something like:
```
  "dependencies": {
    "@tsed/passport": "5.44.13",
    "passport-jwt": "4.0.0",
  },
  "devDependencies": {
    "@types/passport-jwt": "3.0.3",
    "passport": "0.4.1",
  }
```
in my packages.json to make passportJS work also with debugging and `yarn start`

This time I have tried to add '@types/typeorm' to devDependencies, but it was not working:
```
$ npm install @types/typeorm --save-dev
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@types%2ftypeorm - Not found
npm ERR! 404
npm ERR! 404  '@types/typeorm@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

Can you help me to make typeorm work with VSCode debugging?

**Comment by Romakita** [comment]
Hello @deaktomi 
The guide have that:
```
npm install --save @tsed/typeorm
npm install --save typeorm
```
There is no `@types/typeorm` in theses command. Try @tsed/typeorm instead ;)
See you
Romain

**Comment by deaktomi** [comment]
I have installed both of them. I can verify that both of them is in the dependencies:
```
"dependencies": {
  "@tsed/typeorm": "5.44.13",
  "typeorm": "0.2.24"
},
```
But as I say the debugging is still not working

**Comment by deaktomi** [comment]
I believe I have solved the issue. Something weird happened:

- I just deleted the node_modules folder
- Closed VSCode
- Run `yarn install` from terminal

Then I didn't receive the error message anymore, but the debugger silently stopped.

In the DEBUG CONSOLE I was this output:
```
C:\Program Files\nodejs\node.exe --inspect-brk=46685 node_modules\typeorm\index.js 
Debugger listening on ws://127.0.0.1:46685/80453bdf-53bd-4175-8755-4bf64955e65d
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
Waiting for the debugger to disconnect...
```
Which was very weird, because it seems like that the debugger didn't start my index.js, but the typeorm's index.js.

I have renamed my index.ts to app.ts and changed the program line in my launch.json from index.ts to app.ts
`"program": "${workspaceFolder}\\src\\app.ts",`
